src/redux/features/observations/utils.ts

```diff
@@ -0,0 +1,202 @@
+import getDateDisplayValue from '@terraware/web-components/utils/date';
```

Comment:
i moved this code in here from observationsSelectors just to keep the selectors code less messy, esp if we need to add more selectors...